mulighetsrommet-api/src/main/resources/application-dev.yaml

```diff
@@ -60,3 +60,7 @@ app:
   amtEnhetsregister:
     url: http://amt-enhetsregister.amt
     scope: api://dev-gcp.amt.amt-enhetsregister/.default
+
+  msGraphConfig:
+    url: https://graph.microsoft.com
```

Comment:
Hvordan funker dette for dev vs prod? Det håndteres automatisk fra microsoft sin side fordi alt vi opererer med er objekter med unike id'er?

Reply:
Jeg skal få det bekreftet med Gård, men det er slik oppsettet er i poao-tilgang så jeg antar det er som du sier, unike id'er uavhengig av miljø.

Reply:
Den bruker tokenet som sendes med i Authorization-headeren til å skille mellom tenants